src/coprocessor/tracker.rs

```diff
@@ -350,20 +350,24 @@ impl<E: Engine> Tracker<E> {
             false
         };
 
-        tls_collect_query(
-            region_id,
-            peer,
-            start_key.as_encoded(),
-            end_key.as_encoded(),
-            reverse_scan,
-        );
-        tls_collect_read_flow(
-            self.req_ctx.context.get_region_id(),
-            Some(start_key.as_encoded()),
-            Some(end_key.as_encoded()),
-            &total_storage_stats,
-            self.buckets.as_ref(),
-        );
+        // only collect metrics for select and index, exclude transient read flow such
+        // like analyze and checksum.
+        if self.req_ctx.tag == ReqTag::select || self.req_ctx.tag == ReqTag::index {
```

Comment:
Are store and region statistics updated here?

Reply:
yes